WD-Firmware/rtos/rtos_core/freertos/Source/include/portable.h

```diff
@@ -52,6 +52,11 @@ must be set in the compiler's include path. */
 	#include "portmacro.h"
 #endif
 
+extern int rtosalStartScheduler(void);

```

Comment:
those functions and define should not be here. AL should use FreeRTOS and not the opposite way.... double check

Reply:
Need to look at it again. 
example: FreeRTOS, Tasks.c , vTaskStartScheduler function calls to xPortStartScheduler which is defined in portable.h to rtosalStartScheduler